lib/package/components/avatar.css

```diff
@@ -0,0 +1,26 @@
+@charset "UTF-8";
+
+.avatar {
+  border-radius: 50%;
+  overflow: hidden;
+  position: relative;
+}
+.avatar img {
```

Comment:
Everything looks good, nice job. Just one suggestion here, give at least a line space between two declarations. Like - 
```css
.avatar {
}

.avatar img {
}
```

Reply:
> Everything looks good, nice job. Just one suggestion here, give at least a line space between two declarations. Like -
> 
> ```css
> .avatar {
> }
> 
> .avatar img {
> }
> ```

Yes, I usually do that. Thanks for noting.